### PR TITLE
Am cleanup

### PR DESCRIPTION
**Pull request by gpsanant**
various comment and code cleanup

- fixed a bug on line 219

@shrimalmadhur can you verify these view functions aren't needed?

### CODE REVIEW

File: src/contracts/core/AllocationManager.sol [hunk 1]
```diff
@@ -222,25 +214,25 @@ contract AllocationManager is
                     PendingFreeMagnitude storage pendingFreeMagnitude =
                         _pendingFreeMagnitude[operator][strategies[i]][index];
 
-                    // Reached pendingFreeMagnitude/deallocation that is completable and not within slashability window,
+                    // Reached deallocation that is completable and, therefore, not slashable,
                     // therefore older deallocations will also be completable. Since this is ordered by completableTimestamps break loop now
-                    if (pendingFreeMagnitude.completableTimestamp >= uint32(block.timestamp)) {
+                    if (block.timestamp >= pendingFreeMagnitude.completableTimestamp) {
```
Comment (shrimalmadhur) [inline src/contracts/core/AllocationManager.sol:219]: oh this is an interesting use - I should test.

**Review by shrimalmadhur — APPROVED** [review]

File: src/contracts/core/AllocationManager.sol (hunk 1 above)
Comment (gpsanant) [reply src/contracts/core/AllocationManager.sol:219]: whoops i just rewrote this and didn't realize i fixed a bug

**Review by 0xClandestine — APPROVED** [review]
lgtm

**Comment by 0xClandestine** [comment]
forge fmt

**Pull request merged**